src/parser/transform/statement/transform_alter_sequence.cpp

```diff
@@ -17,12 +18,18 @@ unique_ptr<AlterStatement> Transformer::TransformAlterSequence(duckdb_libpgquery
 		throw InternalException("Expected an argument for ALTER SEQUENCE.");
 	}
 
+	uint32_t used = 0;
```

Comment:
Can we turn this into an `unordered_set<SequenceInfo>` instead?

Reply:
Done